packages/client-config/API.md

```diff
@@ -48,7 +60,100 @@ export type AuthClientConfig = {
 };
 
 // @public
-export type ClientConfig = Partial<AnalyticsClientConfig & AuthClientConfig & GeoClientConfig & GraphqlClientConfig & NotificationsClientConfig & StorageClientConfig & PlatformClientConfig & CustomClientConfig>;
+interface AWSAmplifyGen2BackendOutputs {
+    $schema?: string;
+    analytics?: {
+        aws_region: string;

```

Comment:
This is the proposal from the product. I guess to keep the existing case from the legacy config?

Reply:
I'd not use legacy config as precedent here.
I think we should establish what's desired casing for properties across backend and client libraries (which is probably totally inconsistent today) and apply it here as well.

Reply:
@josefaidt thoughts ?

Reply:
I originally had SCREAMING_SNAKE_CASE since these values are more or less readonly constants/env vars, and later pivoted to lowercase partly to keep the existing case